doc/man3/EVP_EncryptInit.pod

```diff
@@ -278,6 +295,11 @@ based on the cipher context. The EVP_CIPHER can provide its own random key
 generation routine to support keys of a specific form. B<Key> must point to a
 buffer at least as big as the value returned by EVP_CIPHER_CTX_key_length().
 
+EVP_CIPHER_do_all_ex() traverses all messages digest implemented by all
+activated providers in the given library context I<libctx>, and calls the
+given functions I<fn> with the implementation method and the given I<arg>
+as argument, for each of the implementations.
```

Comment:
Does this read better with the _for each of the implementations_ placed after the _and_ (still as a comma bracket phrase).

Reply:
So should that be:

> ... and for each of the implementations calls the given functions I\<fn\> with the implementation method and the given I\<arg\> as argument, for each of the implementations.

or:

> ... and for each of the implementations, calls the given functions I\<fn\> with the implementation method and the given I\<arg\> as argument, for each of the implementations.

The difference is a comma.  This is a kind of detail where my english fails me...

Reply:
I'm gonna assume the latter for now.

Reply:
yes, add the comma.  It's an appositive phrase.

Reply:
Thanks.  I ended up doing exactly that.